File: Src/java/cql-to-elm/src/jvmTest/java/org/cqframework/cql/cql2elm/operators/AggregateTest.java
package org.cqframework.cql.cql2elm.operators;

import java.io.IOException;
import org.cqframework.cql.cql2elm.TestUtils;
import org.junit.jupiter.api.Test;

class AggregateTest {

    @Test
    void aggregate() throws IOException {
        TestUtils.runSemanticTest("OperatorTests/Aggregate.cql", 0);
    }
}

File: Src/java/cql-to-elm/src/jvmTest/kotlin/org/cqframework/cql/cql2elm/operators/AggregateTest.kt
package org.cqframework.cql.cql2elm.operators

import java.io.IOException
import org.cqframework.cql.cql2elm.TestUtils.runSemanticTest
import org.junit.jupiter.api.Test

internal class AggregateTest {
    @Test
    @Throws(IOException::class)
    fun aggregate() {
        runSemanticTest("OperatorTests/Aggregate.cql", 0)
    }
}
